HtmlDiff/Diff.cs

```diff
@@ -65,6 +65,23 @@ public class HtmlDiff
         public bool IgnoreWhitespaceDifferences { get; set; }
 
         /// <summary>
+        /// If greater than 0, the text will be broken into div sections that contain
+        /// the number of lines before and after each change.
+        /// Where changes are closer than the context, a new section is not started, and all content output
+        /// If there are no changes, then a null string is returned.
+        /// </summary>
+        /// <example>LinesContext = 1 // Only the line with the change is returned</example>
+        /// <example>LinesContext = 4 // The line with the change is returned and up to 3 lines before and after</example>
+        public int LinesContext { get; set; }
+
+        /// <summary>
+        /// When using the Context option, the name of the class that is injected into the groupers
+        /// </summary>
+        /// <remarks>If LinesContext is provided, and no ContextSelectionClass is </remarks>
+        /// <example></example>
+        public string ContextSectionClass { get; set; }
+
+        /// <summary>
```

Comment:
I wonder if it would be neater to drop these properties in favour of a dedicated public method to process the summary. 

Example:

```
public string BuildSummary(int numberOfLinesToIncludeAroundChange = 0, string diffContainerCSSClass = "diffsection")
{
  // ...
}
```

This makes it easier for callers to understand the configuration options available to them when building a summary of changes.